pkg/scheduler/algorithm/predicates/max_attachable_predicate_test.go

```diff
@@ -0,0 +1,866 @@
+/*
```

Comment:
i think the name of this file as max_attachable_volume_predicate_test.go would be more helpful.

Reply:
fixed